content/docs/studio/overview.md

```diff
@@ -1,3 +1,12 @@
+<admon>
+
+**We have renamed Views to Projects in Iterative Studio.**
+
+Accordingly, _Views dashboard_ is now called _Projects dashboard_; _View
+settings_ are now called _Project settings_; and so on.
+
+</admon>
+
```

Comment:
Same here.

```suggestion
```

Reply:
Removed. And added a note just before the video on this page, coz the video mentions "Views".